eng/targets/Imports.targets

```diff
@@ -102,10 +102,19 @@
 
   <!-- Rulesets -->
   <PropertyGroup Condition="'$(CodeAnalysisRuleSet)' == ''">
-    <CodeAnalysisRuleSet Condition="'$(IsShipping)' == 'true'">$(RepositoryEngineeringDir)config\rulesets\Shipping.ruleset</CodeAnalysisRuleSet>
-    <CodeAnalysisRuleSet Condition="'$(IsShipping)' != 'true'">$(RepositoryEngineeringDir)config\rulesets\NonShipping.ruleset</CodeAnalysisRuleSet>
+    <CodeAnalysisRuleSet>$(RepositoryEngineeringDir)config\rulesets\Empty.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
 
+  <!-- Global Analyzer Config -->
+  <ItemGroup>
+    <!-- Always include Common.globalconfig -->
+    <EditorConfigFiles Include="$(RepositoryEngineeringDir)config\rulesets\Common.globalconfig" />
+    <!-- Include Shipping.globalconfig for shipping projects -->
+    <EditorConfigFiles Condition="'$(IsShipping)' == 'true'" Include="$(RepositoryEngineeringDir)config\rulesets\Shipping.globalconfig" />
```

Comment:
Tagging @chsienki - I believe including these as [GlobalAnalyzerConfigFiles](https://github.com/dotnet/roslyn/blob/8637000fc35f01a453349147b71d5ecdae8d0bff/src/Compilers/Core/MSBuildTask/Microsoft.Managed.Core.targets#L88-L89) should work and make it more clear that these are global analyzer configs, not editorconfigs.

Reply:
There are two problems with this approach:

1. This repository currently uses a compiler build that does not support this property
2. The conversion of `GlobalAnalyzerConfigFiles` to `EditorConfigFiles` is conditioned on `$(DiscoverGlobalAnalyzerConfigFiles)`:

    https://github.com/dotnet/roslyn/blob/8637000fc35f01a453349147b71d5ecdae8d0bff/src/Compilers/Core/MSBuildTask/Microsoft.Managed.Core.targets#L89

    We would need to remove this condition (and wait for it to apply to the SDK used for this repository) before `GlobalAnalyzerConfigFiles` becomes a viable alternative to `EditorConfigFiles`.

Reply:
Confirmed the property `GlobalAnalyzerConfigFiles` works as expected: [ClassLibrary58.zip](https://github.com/dotnet/roslyn/files/5488984/ClassLibrary58.zip)

Reply:
> Confirmed the property GlobalAnalyzerConfigFiles works as expected

It might work in newer SDKs, but it doesn't work in the version of the SDK and/or compiler currently specified for Roslyn command line builds. 

It also doesn't address the fact that setting unrelated property `DiscoverGlobalAnalyzerConfigFiles` currently disables the manual inclusion of global analyzer config files.

Reply:
`DiscoverGlobalAnalyzerConfigFiles` is not defaulted anywhere as per my knowledge, so the condition should be true by default and current logic should work.

Reply:
> It also doesn't address the fact that setting unrelated property DiscoverGlobalAnalyzerConfigFiles currently disables the manual inclusion of global analyzer config files.

Yes, that seems reasonable - @chsienki I can file a tracking bug for you if you agree.